lib/linkscape/client.rb

```diff
@@ -179,8 +179,19 @@ def anchorMetrics(*args)
       Linkscape::Request.run(@options.merge(options))
     end
 
+    def metadata(*args)
+      options = Hash === args.last ? args.pop.symbolize_keys : {}
+      command = args.first ? args.shift : options[:command]
```

Comment:
> `command` seems like a bit of an odd name for this parameter.

The [API documentation](http://moz.com/help/guides/moz-api/mozscape/api-reference/metadata) actually refers to `:last_update`, `:next_update`, and `:index_stats` as "commands". I'm rethinking it - the name does seem particular to the Mozscape internals while this method is dealing with a specific API call anyway. I'll rename the variable and option key to `:type` tomorrow.

FYI, these as already referred to as "types" in the gem README.